membership.lua

```diff
@@ -377,14 +381,27 @@ local function wait_ack(uri, ts, timeout)
 
         for _, ack in ipairs(_ack_cache) do
             if ack.dst == uri and ack.ts == ts then
-                return true
+                return true, ack
             end
         end
     until (now >= deadline) or not _ack_trigger:wait(tonumber(deadline - now) / 1.0e6)
 
     return false
 end
 
+local function _get_clock_delta(ack_data)
+    checks('table')
+    local ack_ts = tonumber(ack_data.ats)
+    local recv_ts = tonumber(ack_data.rts)
+    local start_ts = tonumber(ack_data.ts)
+
+    if ack_ts == nil or recv_ts == nil or start_ts == nil then
+        return nil
+    end
+
+    return math.abs(ack_ts - (recv_ts + start_ts) / 2)
```

Comment:
No need for `abs`. Think of it the same way you think about time zones - GMT+3, GMT-6